### PR TITLE
Release v0.10.2

### LINKED ISSUE #1032: Release 0.10.2
Release v0.10.2

Target: July 2023

Aim is to fix any issues for Optimas. See https://github.com/optimas-org/optimas/pull/88.

Before release:

- Platform testing libE/Optimas
- Optimas custodians approval

Prep setup.py and spack package
- Should setuptools be removed from setup.py requires
- Should pyyaml and tomli be required 
   - also pyyaml currently listed as required and as extra.
   - and make sure listed dependencies correct - https://libensemble.readthedocs.io/en/main/advanced_installation.html
- Use new recommended way for setup.py to get version from version.py
- Make setup.py long description use the README instead of docstring.

Changes:

- [x] Workflow dir fixes #1017 / #1020
- [x] https://github.com/Libensemble/libensemble/pull/1026

### PR DESCRIPTION
**Pull request by shuds13**
Closes #1032

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -8,6 +8,26 @@ GitHub issues are referenced, and can be viewed with hyperlinks on the `github r
 
 .. _`github releases page`: https://github.com/Libensemble/libensemble/releases
 
+Release 0.10.2
+--------------
+
+:Date: July 20, 2023
+
+* Fix issues with workflow dirs:
+  * Ensure relative paths are interpreted from where calling script is run. #1020
```
Comment (jmlarson1) [inline CHANGELOG.rst:17]: Is it from "from where libEnsemble is being run" or "where the script is located"? (What if a user calls a script in a directory than where they are sitting?)

Comment (shuds13) [reply CHANGELOG.rst:17]: A relative path is telling libEnsemble where the script is located.

File: setup.py [hunk 2]
```diff
@@ -15,6 +15,7 @@
 """
 
 from pathlib import Path
+
```
Comment (shuds13) [inline setup.py:18]: Whats the purpose of this line?

**Review by jmlarson1 — APPROVED** [review]

File: setup.py (hunk 2 above)
Comment (jmlarson1) [reply setup.py:18]: `isort` includes it.

File: CHANGELOG.rst (hunk 1 above)
Comment (shuds13) [reply CHANGELOG.rst]: Going with "Ensure relative paths are interpreted from where libEnsemble is run"

File: setup.py (hunk 2 above)
Comment (jlnav) [reply setup.py:18]: of importing Path? it's used on line 52

**Review by jlnav — APPROVED** [review]

**Pull request merged**